README.md

```diff
@@ -1,14 +1,24 @@
-![development-status](https://badgen.net/badge/development-status/alpha/yellow?icon=github)
+# Ably Collaborative Spaces SDK
 
-# Ably
+<p align="center">
+  <a href="">
+    <img src="https://badgen.net/badge/development-status/alpha/yellow?icon=github" alt="Development status"   />
+  </a>
+  <a href="">
+    <img src="https://github.com/ably-labs/spaces/actions/workflows/dev-ci.yml/badge.svg?branch=main" alt="CI status"   />
+  </a>
+    <a href="">
+    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
+  </a>
+</p>
 
-_[Ably](https://ably.com/) is the platform that powers realtime experiences at scale, including live chat, data broadcast, notifications, audience engagement and collaboration._
+The [Ably](https://ably.com) Collaborative Spaces SDK enables you to implement realtime collaborative features in your applications. 
 
-## Collaborative Spaces SDK
+![Example collaboration GIF](/docs/images/collab.gif)
 
-The Collaborative Spaces SDK enables you to implement realtime collaborative features in your applications. Rather than having to coordinate resources on calls, or send documents and spreadsheets back and forth using a combination of tools, having in-app realtime collaboration features has proven to boost productivity in remote workplaces.
+> The Collaborative Spaces SDK is currently under development. If you are interested in being an early adopter and providing feedback then you can [sign up](https://go.ably.com/spaces-early-access) for early access and are welcome to [provide us any feedback](https://go.ably.com/spaces-feedback).
```

Comment:
Let's move it to the end of the section. So we get to describe the project first, before sharing the caveat.

Reply:
Created a new 'status' section at the bottom of the README. I think this makes more sense, but let me know if you'd prefer it more prominent.

Reply:
usually projects have this fairly prominent; providing an in-progress library, where each release could have breaking changes is very different to a library that has at least one stable release

Reply:
Happy to move it up before 'quickstart' - wasn't sure if the alpha status tag was enough

Reply:
FWIW, I did mean at the end of the section before quickstart, not at the end of the readme. @dpiatek @m-hulbert